contrib/bcachefs-tools/patches/makefile.patch

```
@@ -23,28 +23,29 @@ index 0e5cf6e..b55dd97 100644
 -	$(LN) -sfr $(DESTDIR)$(ROOT_SBINDIR)/bcachefs $(DESTDIR)$(ROOT_SBINDIR)/mkfs.fuse.bcachefs
 -	$(LN) -sfr $(DESTDIR)$(ROOT_SBINDIR)/bcachefs $(DESTDIR)$(ROOT_SBINDIR)/fsck.fuse.bcachefs
 -	$(LN) -sfr $(DESTDIR)$(ROOT_SBINDIR)/bcachefs $(DESTDIR)$(ROOT_SBINDIR)/mount.fuse.bcachefs
--
--	sed -i '/^# Note: make install replaces/,$$d' $(DESTDIR)$(INITRAMFS_HOOK)
--	echo "copy_exec $(ROOT_SBINDIR)/bcachefs /sbin/bcachefs" >> $(DESTDIR)$(INITRAMFS_HOOK)
 +	$(INSTALL) -d $(DESTDIR)$(ROOT_SBINDIR)
 +	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/man/man8
 +	$(INSTALL) -d $(DESTDIR)$(INITRAMFS_DIR)/scripts/local-premount
 +	$(INSTALL) -d $(DESTDIR)$(INITRAMFS_DIR)/hooks
 +	$(INSTALL) -d $(DESTDIR)$(PKGCONFIG_UDEVRULESDIR)
-+	$(INSTALL) -m0755 bcachefs $(DESTDIR)$(ROOT_SBINDIR)
-+	$(INSTALL) -m0644 bcachefs.8 $(DESTDIR)$(PREFIX)/share/man/man8/
++	$(INSTALL) -m0755 target/*/release/bcachefs $(DESTDIR)$(ROOT_SBINDIR)
```

not sure of a better way to do this except passing the triple via makefile arg